pkg/koordlet/runtimehooks/proxyserver/server.go

```diff
@@ -17,26 +17,34 @@ limitations under the License.
 package proxyserver
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"net"
+	"path/filepath"
 	"syscall"
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
 	"k8s.io/klog/v2"
 
 	runtimeapi "github.com/koordinator-sh/koordinator/apis/runtime/v1alpha1"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/runtimehooks/hooks"
+	"github.com/koordinator-sh/koordinator/pkg/runtimeproxy/config"
 )
 
 type Options struct {
-	Network string
-	Address string
+	Network        string
+	Address        string
+	FailurePolicy  config.FailurePolicyType
+	ConfigFilePath string
 }
 
 type Server interface {
 	Setup() error
 	Start() error
 	Stop()
+	RegisterPluginServerToProxy() error
```

Comment:
how about `Register()` or `RegisterToProxy`

Reply:
Sure, `Register()` would be better